src/style/colours.rs

```diff
@@ -386,6 +416,30 @@ impl render::SizeColours for Colours {
     fn minor(&self)   -> Style { self.size.minor }
 }
 
+impl render::TimeColours for Colours
+{
+  fn stamp_age (&self, age: i64) -> Style {
+      if self.time_scale {
+          if age < 60 * 60 * 24 {
+              self.date.time_today
+          } else if age < 60 * 60 * 24 * 2 {
+              self.date.time_yesterday
+          } else if age < 60 * 60 * 24 * 7 {
+              self.date.time_week
+          } else if age < 60 * 60 * 24 * 30 {
+              self.date.time_month
+          } else if age < 60 * 60 * 24 * 365 {
+              self.date.time_year
+          } else {
+              self.date.time_past
+          }
+      }
+      else {
+          self.date.time_today
+      }
```

Comment:
I think it would be better to use named colors, as I did here:
https://github.com/ogham/exa/pull/370